octobot_trading/data/order.py

```diff
@@ -248,12 +249,18 @@ def get_currency_and_market(self) -> (str, str):
     def get_total_fees(self, currency):
         return get_fees_for_currency(self.fee, currency)
 
+    def is_open(self):
+        return self.status in [OrderStatus.OPEN, OrderStatus.PARTIALLY_FILLED]
```

Comment:
I think it should be a set instead of a list

Reply:
Why ?